packages/calcite-components/src/components/block/block.scss

```diff
@@ -170,12 +191,14 @@ calcite-handle {
 
 .content {
   @apply animate-in flex-1 relative min-h-0;
-  padding-block: var(--calcite-block-padding, var(--calcite-spacing-sm));
-  padding-inline: var(--calcite-block-padding, var(--calcite-spacing-md));
+  padding-block: var(--calcite-spacing-sm);
+  padding-inline: var(--calcite-spacing-md);
 }
 
 .content-start {
-  @apply text-color-3 flex items-center;
+  @apply flex items-center;
+
+  color: var(--calcite-block-content-start-color, var(--calcite-color-text-3));
```

Comment:
`--calcite-block-text-color` represents the existing design pattern and we don't want to add extra tokens unless specifically asked by our users